src/Gui/DlgAddPropertyVarSet.cpp

```diff
@@ -101,19 +101,30 @@ void DlgAddPropertyVarSet::initializeGroup()
     comboBoxGroup.setEditText(QString::fromStdString(groupNamesSorted[0]));
 }
 
+void DlgAddPropertyVarSet::getSupportedTypes(std::vector<Base::Type>& types)
+{
+    std::vector<Base::Type> proptypes;
+    Base::Type::getAllDerivedFrom(Base::Type::fromName("App::Property"), proptypes);
+    std::copy_if(proptypes.begin(), proptypes.end(), std::back_inserter(types), [](const Base::Type& type) {
+        return type.canInstantiate();
+    });
+    std::sort(types.begin(), types.end(), [](Base::Type a, Base::Type b) {
+        return strcmp(a.getName(), b.getName()) < 0;
+    });
+}
+
 void DlgAddPropertyVarSet::initializeTypes()
 {
     auto paramGroup = App::GetApplication().GetParameterGroupByPath(
             "User parameter:BaseApp/Preferences/PropertyView");
-    auto lastType = Base::Type::fromName(
+   auto lastType = Base::Type::fromName(
```

Comment:
```suggestion
    auto lastType = Base::Type::fromName(
```

Reply:
Thanks, my difftool didn't show this for some reason, just fixed it. The changes of `Baseclass.h` are from Werner's commit in PR #15191 (that I recommend to merge first and that I base my changes on), so effectively this PR only makes changes to the VarSet dialog.